rollup.config.js

```diff
@@ -35,7 +35,7 @@ export default {
     }),
     uglify({
       compress: {
-        warnings: false,
+        // warnings: false,
```

Comment:
Is there a reason this line needs to commented out, or was this from debugging?

Reply:
I needed to disable it in order to have my local tests running, but I just reverted that change in https://github.com/gregnb/mui-datatables/pull/589/commits/09917f9b98ef47da6f3a5aea513cafe5194e1c9c.